index.js

```diff
@@ -182,7 +188,26 @@ function numThreadHangs() {
 }
 
 function numEventLoopLags() {
-  let snapshot = Services.telemetry.getHistogramById("EVENTLOOP_UI_ACTIVITY_EXP_MS").snapshot();
+  try {
+    var snapshot = Services.telemetry.getHistogramById("EVENTLOOP_UI_ACTIVITY_EXP_MS").snapshot();
+  } catch (e) { // histogram doesn't exist, the Firefox version is likely older than 45.0a1
+    return null;
+  }
+  let result = 0;
+  for (let i = 0; i < snapshot.ranges.length; ++i) {
+    if (snapshot.ranges[i] > 50) {
```

Comment:
Wait a minute... these aren't governed by the same threshold as threadHangs? I thought they were.